config/mediaflux.yml

```diff
@@ -26,7 +26,7 @@ development:
 test:
   api_root_ns: '/td-test-001'
   api_transport: 'http'
-  api_host: '0.0.0.0'
+  api_host: 'test.mediaflux.com'
```

Comment:
Would it make sense to use an `example.com` domain to make sure it never tries to contact a real MediaFlux server? `test.mediaflux.com` doesn't resolve now, but they could register it if they wanted.